packages/libs/sdk/tsconfig.lib.json

```diff
@@ -6,15 +6,14 @@
     "target": "ES2020",
     "module": "ES2020",
     "outDir": "dist",
-    "rootDir": "src"
+    "rootDir": "."
   },
-  "include": ["**/*.ts"],
+  "include": ["package.json", "src/**/*.ts"],
   "exclude": [
     "jest.config.ts",
     "**/*.spec.ts",
     "**/*.test.ts",
     "webpack.config.ts",
-    "src/spec/testSetup/**.*",
```

Comment:
This isn't a file anymore (also, since we are requiring `src/**/*.ts`, this exclude list isn't really doing much, but I think it's still good to be explicit here to be extra cautious about test files not being pulled in